lib/fog/storage/google_xml/models/file.rb

```diff
@@ -118,7 +118,7 @@ def save(options = {})
 
         def url(expires, options = {})
           requires :key
-          collection.get_http_url(key, expires, options)
+          collection.get_https_url(key, expires, options)
```

Comment:
Hmm, we may want to support the `:scheme` option after all.

Reply:
Perhaps not, given `get_object_url` is deprecated in favor of `get_object_https_url`.